### PR TITLE
Remove Trusty support

### PR DESCRIPTION
**Pull request by leandro-lucarella-sociomantic**
Remove trusty support from all projects. Trusty is already more than 3 years old and the latest LTS (Xenial) has already been out there for more than a year, so it should be enough to maintain the latest LTS only now.

### CODE REVIEW

**Comment by nemanja-boric-sociomantic** [comment]
```
make: *** No rule to make target 'ci/run.sh'.  Stop.
```

File: .travis.yml [hunk 1]
```diff
@@ -19,16 +19,12 @@ env:
...
+        - DMD=dmd1 DIST=xenial F=production AFTER_SCRIPT=1
+        - DMD=dmd1 DIST=xenial F=devel
+        - DMD=dmd-transitional DIST=xenial F=production
+        - DMD=dmd-transitional DIST=xenial F=devel
 
-install: beaver install
+install: beaver dlang install
 
-script: BEAVER_DOCKER_VARS="DC" beaver run ci/run.sh
+script: BEAVER_DOCKER_VARS="F" beaver make ci/run.sh
```
Comment (nemanja-boric-sociomantic) [inline .travis.yml]: This is wrong, should be `beaver dlang make`.

Comment (leandro-lucarella-sociomantic) [reply .travis.yml]: woopsy!

**Comment by leandro-lucarella-sociomantic** [comment]
Now errors while testing. Maybe is the same problem as in the DHT node?

**Comment by nemanja-boric-sociomantic** [comment]
No, I think it's because the tests are now running in parallel, using the same sandbox directory:

```
[turtle    ] turtle.Exception.TurtleException: `rm -rf /home/travis/build/sociomantic-tsunami/dlsnode/build/devel/tmp/sandbox-dlsnode/` has failed (./submodules/turtle/src/turtle/env/Shell.d:61)
```

**Comment by nemanja-boric-sociomantic** [comment]
Fixed in https://github.com/sociomantic-tsunami/dlsnode/pull/10, please rebase.

**Comment by leandro-lucarella-sociomantic** [comment]
Updated :crossed_fingers:

**Comment by leandro-lucarella-sociomantic** [comment]
Mmmm, still crashing tests, core dumping and all kind of madness :boom:

**Comment by nemanja-boric-sociomantic** [comment]
Ah, I need to listen on the separate ports now !

**Comment by nemanja-boric-sociomantic** [comment]
May I push to this branch?

**Comment by nemanja-boric-sociomantic** [comment]
Replaced with #11 (I had no permissions to push to this branch)

**Pull request closed without merging**